pkg/registries/registries_test.go

```diff
@@ -9,31 +9,69 @@ import (
 	"testing"
 
 	"github.com/BurntSushi/toml"
-	"github.com/containers/image/pkg/sysregistriesv2"
-	"github.com/containers/image/types"
+	"github.com/containers/image/v5/pkg/sysregistriesv2"
+	"github.com/containers/image/v5/types"
 	apioperatorsv1alpha1 "github.com/openshift/api/operator/v1alpha1"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	"k8s.io/apimachinery/pkg/util/diff"
 )
 
-func TestScopeMatchesRegistry(t *testing.T) {
+func TestScopeIsNestedInsideScope(t *testing.T) {
 	for _, tt := range []struct {
 		scope, reg string
```

Comment:
```suggestion
		subScope, superScope string
```
\+ update the comments as well.

Reply:
Fixed